editor/src/document/document_file.rs

```diff
@@ -413,6 +413,14 @@ impl DocumentMessageHandler {
 		self.current_identifier() == self.saved_document_identifier
 	}
 
+	pub fn saved(&mut self, state: bool) {
+		if state {
```

Comment:
It's unclear to me what a boolean called `state` means.

Reply:
Might be worth changing `state` to `is_saved`